ANDROIDCOREFUNCTIONALITY.md

```diff
@@ -0,0 +1,105 @@
+# OpenAthena Android Core Functionality
+Core Casting Functionality within OA Android is defined by 3 main classes: MetaDataExtractor, DEMParser, and TargetGetter.
+These classes are managed by the AthenaApp and MainActivity classes.

```

Comment:
The association here presents a bit of a false equivalency. AthenaApp is a singleton for managing persistent app state, while MainActivity, DEM Activity, FullscreenSelectionActivity, etc. extend the AthenaActivity abstract class. AthenaActivity manages the behavior of subclasses much more than AthenaApp